openpype/hosts/maya/api/workfile_template_builder.py

```diff
@@ -244,13 +233,36 @@ def get_placeholder_options(self, options=None):
         return self.get_load_plugin_options(options)
 
     def post_placeholder_process(self, placeholder, failed):
-        """Hide placeholder, add them to placeholder set
-        """
+        """Hide placeholder, add them to placeholder set"""
         node = placeholder.scene_identifier
 
         cmds.sets(node, addElement=PLACEHOLDER_SET)
         cmds.hide(node)
         cmds.setAttr(node + ".hiddenInOutliner", True)
+        model = cmds.ls(assemblies=True)
+
+        filtered_model = []
+        exclude_list = ["persp", "top", "front", "side", "root", "wip"]
+
+        for element in model:
+            if not any(exclude_elem in element for exclude_elem in exclude_list):
```

Comment:
This will fail to detect the default cameras if they were parented to something since they wouldn't be assemblies.
You might be better of doing the `cmds.camera(cam, query=True, startupCamera=True)` query on the cameras to filter.

Also, instead of filtering the cameras you might be better of just searching for only the visible shapes? `cmds.ls(type=("mesh", "nurbsCurve", "nurbsSurface"), visibleOnly=True)` (even though that doesn't account for shapes that become visible over time?

Reply:
it's visible=True not visbleOnly, this flag does not exist. Thanks for your advice they work well !

Reply:
> it's visible=True not visbleOnly, this flag does not exist.

Thanks, must've mixed it up with another flag or command. Anyway, it was just intended to be an idea - not sure if that's the best workflow either.